parley/src/layout/line/mod.rs

```diff
@@ -88,6 +107,21 @@ impl LineMetrics {
     }
 }
 
+#[derive(Clone)]
+pub enum LayoutItem2<'a, B: Brush> {
```

Comment:
I'd suggest calling this `LayoutItem` and the other type `LayoutItemData` which is the general naming pattern used by other "duplicate" types in this crate.

Reply:
I reckon we ought to consider changing this naming pattern to something a bit more intuitive, but for now this sounds good :)

Reply:
Ok, I've actually gone with `PositionedLayoutItem` here.

Reply:
I’m okay with a better naming pattern as long as it’s consistent and the public API has the nicer names (which was my goal here).